src/content/blog/2024-08-22-the-ugrc-api-address-location-lane.mdx

```diff
@@ -0,0 +1,45 @@
+
+## The foundational data
+
+The UGRC provides authoritative GIS data for Utah through the [SGID](/products/sgid/sgid-index/). By aggregating data from local governments and standardizing it, we are able to offer this comprehensive statewide resource. Our address locating endpoint provides excellent address match rates because of the collaborative efforts of our local address authority partners and their dedicated work. Together, we maintain two foundational datasets that power the UGRC API -- [address points](/products/sgid/location/address-points/) and [road centerlines](/products/sgid/transportation/road-centerlines/). The [zip codes](/products/sgid/boundaries/zip-codes/) and [address systems](/products/sgid/location/address-system-quadrants/) participate and are worth mentioning but we won't be discussing them further.
+
+The road centerlines and address point data is regularly updated, with the "Big 5" counties (Salt Lake, Utah, Davis, Weber, Washington) providing monthly updates. While the other counties provide updates on a less frequent schedule, the "Big 5" cover approximately 80% of Utah's population, ensuring great coverage in densely populated areas. Stay informed about the latest updates by following the [SGID Update](/blog/categories/sgid-updates/) blog category.
+
+## The data limitations
+
+While the UGRC API address locating endpoint is a powerful service, it's important to be aware of its limitations. The accuracy of our data depends on the quality and timeliness of the data provided by our address authority partners. In rural counties, where resources and funding may be limited, data gaps exist. This is most pronounced in counties like Juab, Piute, Uintah, and Sevier. We are working hard to improve the data quality, especially in these areas, but it's a work in progress. These data characteristics make it problematic to solely use our API for address validation for mail delivery, 911 response, or voter registration types of questions.
+
+## Road centerlines versus address points
+
+Road centerlines and address points are two different datasets that work together to provide the best address location match rates. By default, both datasets are used to provide the solution and address points have priority in the case of a tie. The UGRC API provides which dataset solved your request in the response and you are able to specify which data participates in the match using the `locator` optional input. This is important since the road centerlines are used to determine the location of an address along a road segment, while the address points are used to determine the exact location of an address. By using both datasets together, you will get the best match rate our data can provide. But a match on the road centerlines has a different meaning than a match on the address points. But if you are zooming, routing, extracting data, or looking for a generalized location for analysis a match on the road centerlines is a great option.
```

Comment:
do you want to say anything along the lines that we strive to find a match in the address points first? aka: if we get 100% match in both datasets, we go with the address point.

Reply:
> and address points have priority in the case of a tie.

Reply:
good point. nice.